internal/action/command.go

```diff
@@ -1073,8 +1073,8 @@ func (h *BufPane) openTerm(args []string, newtab bool) {
 		h.AddTab()
 		id = MainTab().Panes[pane].ID()
 	} else {
-		for i, p := range h.tab.Panes {
-			if p.ID() == h.ID() {
+		for i, p := range MainTab().Panes {
+			if p.IsActive() {
```

Comment:
Another commit message saying something completely obvious about what the change is doing, instead of explaining why it is doing that...

If there is nothing to explain (i.e. if it's just cosmetics and that's it), that's fine, but why need to extra say something completely obvious?

Reply:
Removed from commit message.